src/intents.cs

```diff
@@ -9099,5 +9099,38 @@ interface INVoiceShortcutCenter {
 	//	[Export ("deferredLocalizedIntentsStringWithFormat:fromTable:arguments:", IsVariadic = true)]
 	//	string DeferredLocalizedIntentsStringWithFormat (string format, [NullAllowed] string table, IntPtr arguments);
 	//}
+
+	[Watch (6,0), NoMac, iOS (13,0)]
+	[Native]
+	public enum INCallAudioRoute : long
+	{
+		Unknown = 0,
+		Speakerphone,
+		Bluetooth,
+	}
+
+	[Watch (6,0), NoMac, iOS (13,0)]
+	[BaseType (typeof (INIntent))]
+	interface INStartCallIntent
```

Comment:
add a `[DisableDefaultCtor]` since
* there's a designated initializer that does not accept `null`, e.g. for `audioRoute` and
* there's no setter on the same property to make it non-null
That makes it unlikely that `init` creates a valid instance